src/lib/decisionsCallbackStorage.js

```diff
@@ -0,0 +1,12 @@
+const decisionsReceivedEventTriggers = [];
+
+module.exports = {
+  add(trigger) {
+    decisionsReceivedEventTriggers.push(trigger);
+  },
+  triggerEvent(decisions) {
+    decisionsReceivedEventTriggers.forEach(trigger => {
```

Comment:
Should we try catch those callbacks since they are custom code?

Reply:
`trigger` is part of the event definition, we completely control it. This is not something that is configured by the customer.